release-notes/opendistro-for-elasticsearch-security.release-notes-1.10.1.0.md

```diff
@@ -23,7 +23,7 @@ Supported Elasticsearch version 7.9.0
 
 ### Maintenance
 
-* Support ES 7.9.0 ([#661](https://github.com/opendistro-for-elasticsearch/security/pull/661))
```

Comment:
Please keep support for 7.9.0 PR as an item in the list. Changes in that PR impact both API and behavior.